modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -907,17 +913,17 @@ export default {
       if (item.options.toggle) {
         if (this.trayItemState[item.name] && item.options.closeLabel) {
           return {
-            content: item.options.closeLabel,
+            content: item.options.closeTooltip,
```

Comment:
`closeTooltip`/`openTooltip` doesn't seem right. That's implying this is always going to be opening and closing. Could `tooltip` have an object option, or always be an object, and these be `active/inactiveLabel` properties?